### DISCUSSION TITLE
Welcome to Bach Discussions!

### DISCUSSION THREAD

**Discussion by eddievelasquez** [eddievelasquez/Bach#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by barisaxo** [comment]
Any reason you're using ## instead of x for double sharp?

Might also be worth considering 'simplified symbols' (bb, b,  , #, x) as well as 'unicode symbols'

𝄫 : u+1d12b
♭ : u+266d
♮ : u+266e
♯ : u+266f
𝄪 : u+1d12a

**Reply by eddievelasquez** [reply]
Not a bad idea. I'll look into it. (Sorry for the long delay, have been very busy at work and had this project quite abandoned. I intend to fix this.)
